src/lib/statistics/column_statistics.cpp

```diff
@@ -247,20 +242,12 @@ FilterByColumnComparisonEstimate ColumnStatistics<ColumnDataType>::estimate_pred
   auto right_above_overlapping_ratio = 0.f;
 
   if (std::is_integral<ColumnDataType>::value) {
```

Comment:
Can be also replaced to `if constexpr (std::is_integral_v<ColumnDataType>) {`.
Makes it consistent with the other if condition.

Reply:
Good catch, thanks, I changed it in two more places as well.